modules/kargoBidAdapter.js

```diff
@@ -42,8 +42,8 @@ export const spec = {
       bidResponses.push({
         requestId: bids[adUnitId].bidId,
         cpm: Number(adUnit.cpm),
-        width: adUnit.width,
-        height: adUnit.height,
+        width: bids[adUnitId].sizes[0][0],
```

Comment:
Please confirm that the `sizes` you're referring to here is in your response body. My reading of this code is that bids[] is the original PBJS AdUnit structure. 

First problem is that the pbjs AdUnit.sizes attribute is deprecated in favor of AdUnit.mediaType.banner.sizes. But more importantly, the height and width have to be coming from your bid response, not assuming the first size of the requested list.